ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/fastsync/ImportSyncBlocksStep.java

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright contributors to Hyperledger Besu.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.besu.ethereum.eth.sync.fastsync;
+
+import org.hyperledger.besu.ethereum.ProtocolContext;
+import org.hyperledger.besu.ethereum.core.BlockHeader;
+import org.hyperledger.besu.ethereum.core.BlockImporter;
+import org.hyperledger.besu.ethereum.core.SyncBlockWithReceipts;
+import org.hyperledger.besu.ethereum.eth.manager.EthContext;
+import org.hyperledger.besu.ethereum.eth.sync.ValidationPolicy;
+import org.hyperledger.besu.ethereum.eth.sync.tasks.exceptions.InvalidBlockException;
+import org.hyperledger.besu.ethereum.mainnet.BlockImportResult;
+import org.hyperledger.besu.ethereum.mainnet.ProtocolSchedule;
+
+import java.util.List;
+import java.util.OptionalLong;
+import java.util.concurrent.TimeUnit;
+import java.util.function.Consumer;
+
+import com.google.common.annotations.VisibleForTesting;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ImportSyncBlocksStep implements Consumer<List<SyncBlockWithReceipts>> {
+  private static final Logger LOG = LoggerFactory.getLogger(ImportSyncBlocksStep.class);
+  private static final long PRINT_DELAY = TimeUnit.SECONDS.toMillis(30L);
+
+  private final ProtocolSchedule protocolSchedule;
+  protected final ProtocolContext protocolContext;
+  private final EthContext ethContext;
+  private long accumulatedTime = 0L;
+  private OptionalLong logStartBlock = OptionalLong.empty();
+  private final BlockHeader pivotHeader;
+
+  public ImportSyncBlocksStep(
```

Comment:
The related `ImportBlocksStep` class is only used in tests now and so can be deleted

Reply:
Done